.travis.yml

```diff
@@ -2,10 +2,8 @@
 language: julia
 os:
   - linux
-  - osx
 julia:
   - release
-  - nightly
```

Comment:
Guess the nightly cron job will cover this? Or does the Nanosoldier.jl code only run on the coordinator node with a fixed release Julia?